lib/java/gradle/environment.gradle

```
@@ -71,7 +71,8 @@ dependencies {
     compile "javax.annotation:javax.annotation-api:${javaxAnnotationVersion}"
     compile "org.apache.commons:commons-lang3:3.12.0"
 
-    testCompile "junit:junit:${junitVersion}"
+    testImplementation "org.junit.jupiter:junit-jupiter:${junitVersion}"
+    testImplementation "org.junit.jupiter:junit-jupiter-params:${junitVersion}"
```

This might be redundant. I'm not sure, but I think the `junit-jupiter` artifact includes `junit-jupiter-api`, `junit-jupiter-engine`, and `junit-jupiter-params`. So, you might not need to include the params separately.

```suggestion
    testImplementation "org.junit.jupiter:junit-jupiter:${junitVersion}"
```